packages/simple-types/src/helpers/share-templates-to-groups.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  IItemTemplate,
+  UserSession,
+  getProp,
+  shareItemToGroups
+} from "@esri/solution-common";
+import { maybePush } from "@esri/hub-common";
+
+/**
+ * Given the created templates
+ * @param templates
+ * @param templateDictionary
+ * @param authentication
+ */
+export function shareTemplatesToGroups(
```

Comment:
Curious why we're calling `shareTemplatesToGroups` _only_ from `Notebook` and `SimpleTypes` `postProcess` methods? Is the thinking that different item types may be shared with different groups (or not at all) based on certain criteria?  Or if this should be common to all templates regardless of type, maybe this would make more sense to live in `solution-common` or `solution-deployer` and invoked from `packages/deployer/src/helpers/post-process.ts` instead of individual handler `postProcess` methods that may or may not be implemented?

Reply:
Good question @rweber-esri ! In general - what I've found from our work building a solution system in Hub, is that delegating as much as you can to type-specific processors is better. Almost everything that I was *sure* would be applied to all item types, I've ended up having to hoist back out because one corner case or another proved me wrong.

For this case specifically, Sites have very particular logic around what groups they are shared to, so I pushed this logic into a shared module, and any/all types that need it, can just import and call that fn. I have no strong feelings for where that specific function lives though.